pyproject.toml

```diff
@@ -9,8 +9,8 @@ homepage = "https://meltano.com"
 repository = "https://github.com/meltano/edk"
 keywords = [
     "Meltano",
-    "Meltano SDK",
-    "ELT",
+    "Meltano Extension",
+    "Meltano EDK"
```

Comment:
@aaronsteers @tayloramurphy anything special y'all wanna include here ?  You can also of course just send a PR along out of band to fix up all this.

Reply:
Nothing from my side.

Thanks. 👍

Reply:
Perfect, gonna go ahead and merge. @tayloramurphy if you end up wanting something special just pop an issue and we can update the content for ya.